### ISSUE TITLE
Shareable motion capture / avatar links.

### ISSUE THREAD

**Issue by ngokevin** [supermedium/moonrider#24]
Feature for far future.

1. Record motion capture during gameplay.
2. Checkbox on Victory Screen whether or not to publish (opt-out).
3. Produce shareable links (moonrider.xyz#red-mongoose).
4. Can watch gameplay on any device with spectator camera.

UX to be determined.

**Comment by chrisplatorres** [comment]
This might be a separate issue, but [mixed reality capture](https://youtu.be/YSJRPuOETbI) with [LIV](https://wiki.liv.tv/index.php/Main_Page) seems to be popular for content creators. LIV uses the openVR api though, so it wouldn't be able to be used directly with web.

**Comment by ngokevin** [comment]
Yeah, we know the LIV people, it's pretty tough to set up. We spent a lot of time trying to mesh it with Web, ended up using OBS.

**Comment by ngokevin** [comment]
Need to fix up aframe-motion-capture-components
